test/pg.util.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const metatests = require('metatests');
+const pgUtils = require('../lib/pg.utils');
+
+metatests.case('Test pgUtils.isValidIdentifier', { pgUtils }, {
+  'pgUtils.isValidIdentifier': [
+    ['validIdentifier', true],
+    ['identifierWithSpecialSymbols@!', false],
```

Comment:
I'd suggest reordering tests to have 
all passing (true) tests
all failing (false) tests.